modules/dists/BlockCycDist.chpl

```diff
@@ -627,7 +627,7 @@ proc BlockCyclicDom.dsiBuildArray(type eltType) {
   return arr;
 }
 
-proc BlockCyclicDom.dsiNumIndices return whole.numIndices;
+proc BlockCyclicDom.dsiNumIndices return whole.size;
```

Comment:
Not for this PR, but do we want to change `dsiNumIndices` to `dsiSize`?

Reply:
I wondered about that too, but didn't feel strongly either way (dsiSize would be more symmetrical; dsiNumIndices is more descriptive).